DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/README.md

```diff
@@ -0,0 +1,317 @@
+   
+      *Pin planner from GHRD:*
+
+      ![](assets/pins-from-ghrd.png)
+
+      *Final pin planner configuration:*
+
+      ![](assets/pins-from-design.png)
+
+8. Now add the timing constraints. 
+
+   1. If you are using the Intel® Arria® 10 SX SoC Dev Kit, you can find a timing constraints file for the JTAG interface (jtag.sdc) in the GHRD. This file was added during project creation.
+
+   2. Create a new Synopsis Design Constraints (SDC) file named `add.sdc` and insert a new clock called `i_clk` to match the clock you defined in `add.sv`. Set the period to be 10ns:
+
+      ```
+      set_time_format -unit ns -decimal_places 3
+      create_clock -name i_clk -period 10 [get_ports {i_clk}]
+      ```
+
+   3. Cut the clock paths for asynchronous I/O:
+      
+      ```
+      set_false_path -from [get_ports {reset_button_n}] -to * 
+      set_false_path -from [get_ports {fpga_led}] -to *
+      set_false_path -from * -to [get_ports {fpga_led}]
+      ```
+
+9. Compile the full design by clicking the 'Start Compilation' button in the Intel® Quartus® Prime GUI.
+
+      ![](assets/start-compilation-quartus.png)
+
+10. Copy the generated `add.sof` file to the `system_console` directory.
+
+### Additional Documentation
+- [Intel® Arria® 10 SoC Golden System Reference Design](https://rocketboards.org/foswiki/Documentation/Arria10SoCGSRD) describes a reference design you can use with your Intel® Arria® 10 SX SoC Developer kit.
+- [Intel® Arria 10 SX SoC Development Kit](https://www.intel.com/content/www/us/en/products/details/fpga/development-kits/arria/10-sx.html) describes the Intel® Arria® 10 SX SoC Development kit in greater detail.
+- [Intel® FPGA Software Installation and Licensing](https://www.intel.com/content/www/us/en/docs/programmable/683472/current/faq.html) describes how to license Intel® Quartus® Prime Pro software.
+- [Intel® Quartus® Prime Pro Edition User Guide: Getting Started](https://www.intel.com/content/www/us/en/docs/programmable/683463/current/faq.html) introduces you to the Intel® Quartus® Prime Pro software.
+- [Intel® Quartus® Prime Pro Edition User Guide: Platform Designer](https://www.intel.com/content/www/us/en/docs/programmable/683609/current/faq.html) describes the Intel® Platform Designer software.
+- [Intel® Quartus® Prime Pro Edition User Guide: Programmer](https://www.intel.com/content/www/us/en/docs/programmable/683039/current/programmer-user-guide.html) describes the Intel® Quartus® Prime Pro Programmer software.
+
+## Running the Sample
+
+Use the `test.bat` script in the `system_console` directory to flash the design to your development board, and launch the system console. The included `.tcl` scripts in the `system_console` directory demonstrate how to use the System Console to interact with your IP through the JTAG Avalon Master IP on the FPGA.
+
+If you need to move the design to a different computer, make sure you copy the `system_console` and  directories from the `add-quartus` directory.
```

Comment:
To move the design to a different computer,  copy the \`system_console\` and  directories from the \`add-quartus\` directory.